Imagine/Filter/PostProcessor/JpegOptimPostProcessor.php

```diff
@@ -141,33 +141,33 @@ public function processWithConfiguration(BinaryInterface $binary, array $options
             throw new \RuntimeException(sprintf('Temp file can not be created in "%s".', $tempDir));
         }
 
-        $pb = new ProcessBuilder([$this->jpegoptimBin]);
+        $processArguments = [$this->jpegoptimBin];
 
         $stripAll = array_key_exists('strip_all', $options) ? $options['strip_all'] : $this->stripAll;
         if ($stripAll) {
-            $pb->add('--strip-all');
+            array_push($processArguments, '--strip-all');
```

Comment:
why not `$processArguments[] = '--strip-all';` ?

Reply:
yeah, $processArguments[] = '--strip-all'; makes more sense. I refactored it.